gulpfile.js

```diff
@@ -56,7 +58,15 @@ gulp.task('deploy-assets', function (callback) {
   );
 });
 
+var generated_template_for = function(file) {
+  return file.replace(assetPath + '/templates/', themePath + '/');
+}
+
 gulp.task('clean', function () {
+  glob(assetPath + '/templates/**/*', function(err, files) {
```

Comment:
I'd prefer an atomic structure with 2 tasks `clean-templates` and `clean-dist` both being called by the `clean` task.

Also is there a reason you are using `glob` vs `gulp` here? For consistency, I'd rather use gulp unless there is a good reason.